autorest/retriablerequest_1.x.go

```diff
@@ -0,0 +1,39 @@
+// +build !go1.8
```

Comment:
No this particular file is for versions before 1.8.  The other file will be used in versions 1.8+ (maybe I didn't do such a great job in naming them).